src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -127,21 +147,39 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
 
   protected updated(changedProperties: PropertyValues) {
     super.updated(changedProperties);
-    if (changedProperties.has("entry")) {
-      this._error = undefined;
-      this._name = this.entry.name || "";
-      this._icon = this.entry.icon || "";
-      this._deviceClass =
-        this.entry.device_class || this.entry.original_device_class;
-      this._origEntityId = this.entry.entity_id;
-      this._areaId = this.entry.area_id;
-      this._entityId = this.entry.entity_id;
-      this._disabledBy = this.entry.disabled_by;
-      this._hiddenBy = this.entry.hidden_by;
-      this._device =
-        this.entry.device_id && this._deviceLookup
-          ? this._deviceLookup[this.entry.device_id]
-          : undefined;
+    if (!changedProperties.has("entry")) {
+      return;
+    }
+
+    this._error = undefined;
```

Comment:
All these things will cause a new re-render. That means the changes need to be made in `willUpdate`.